### DISCUSSION TITLE
When creating an API Token, it's always using the first available Scope

### DISCUSSION THREAD

**Discussion by IgnacioNMiranda** [vercel/community#7175] [Help]
### Page to Investigate

https://vercel.com/account/tokens

### Steps to Reproduce

1. Go to https://vercel.com/account/tokens
2. Try to create a token selecting any scope different from the first one available
3. Use the token to make API Calls
4. Receive an error saying something like `Trying to access resource under scope "scope1". You must re-authenticate to this scope or use a token with access to this scope.`

**Comment by amyegan** [comment]
This looks like an error you would see when trying to access `scope1` data when you are actually authorized for a different scope. When you send the request, are you including the `slug` or `teamId` param to specify a team other than the default one?

[a comment by an automated account is not shown]
